### PR TITLE
Progress: alpha order students in progress tab

### PR DESCRIPTION
**Pull request by caleybrock**
Alpha order students by name (previously ordered by order in which they were added to course).
Updated tests for redux action where the sorting takes place.
screen shot 2018-04-13 at 2 16 07 pm

### CODE REVIEW

File: apps/src/templates/sectionProgress/sectionProgressRedux.js [hunk 1]
```diff
@@ -17,13 +17,28 @@ const ADD_STUDENT_LEVEL_PROGRESS = 'sectionProgress/ADD_STUDENT_LEVEL_PROGRESS';
...
 export const addScriptData = (scriptId, scriptData) => ({ type: ADD_SCRIPT_DATA, scriptId, scriptData });
 export const addStudentLevelProgress = (scriptId, studentLevelProgress) => ({
   type: ADD_STUDENT_LEVEL_PROGRESS, scriptId, studentLevelProgress
 });
+export const setSection = (section) => {
+  // Sort section.students by name.
+  const sortedStudents = section.students.sort((studentA, studentB) => {
+    // Ignore case.
+    const studentNameA = studentA.name.toLowerCase();
+    const studentNameB = studentB.name.toLowerCase();
```
Comment (breville) [inline apps/src/templates/sectionProgress/sectionProgressRedux.js]: I'm not familiar with the pros/cons but would it be worth using `localeCompare` instead?

Comment (caleybrock) [reply apps/src/templates/sectionProgress/sectionProgressRedux.js]: Seems better tested and shorter, so I'll go with it 👍

**Comment by Erin007** [comment]
What happens when we make the call to load more students? They'll be sorted into the mix?

**Comment by caleybrock** [comment]
Good question @Erin007 
We currently order the table by the section data, which gets passed in from the top level angular controller. To make this work properly when we paginate, we'll also have to order the data on the server so we send the correct set of progress that corresponds to the first set of students. This will be follow up work when we work on the API.

**Pull request merged**